tests/mypy/outputs/plugin_success.txt

```diff
@@ -1,3 +1,5 @@
 121: error: Unexpected keyword argument "name" for "AddProject"  [call-arg]
 121: error: Unexpected keyword argument "slug" for "AddProject"  [call-arg]
-121: error: Unexpected keyword argument "description" for "AddProject"  [call-arg]
\ No newline at end of file
+121: error: Unexpected keyword argument "description" for "AddProject"  [call-arg]
+139: error: Variable "pydantic.types.StrictUnion" is not valid as a type  [valid-type]
+139: note: See https://mypy.readthedocs.io/en/latest/common_issues.html#variables-vs-type-aliases
```

Comment:
we should rename this file to something less confusing (I know i renamed it before to this name 🙈).

Perhaps 'no_plugin_errors.txt` or something?

Reply:
https://github.com/samuelcolvin/pydantic/pull/2092/commits/76ae8dfd55510bc2eb4b94354ba07b8eb786679a